### PR TITLE
update popup methods addClass / removeClass to return popup instance

### PR DESCRIPTION
**Pull request by ferdicus**
As mentioned in discussion https://github.com/maplibre/maplibre-gl-js/discussions/3972

## Launch Checklist
 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [x] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

**Comment by HarelM** [comment]
Thanks for taking the time to open this PR!
Can you please add a changelog entry and a test to cover the newly added functionality?

**Comment by ferdicus** [comment]
> Thanks for taking the time to open this PR! Can you please add a changelog entry and a test to cover the newly added functionality?

Sorry, have seen this, it's not forgotten :D 

Will come back to this during this week 🙇🏿 

Sorry for the caused inconvenience

**Comment by ferdicus** [comment]
@HarelM, would you kindly have another look

thanks 🙇🏿

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## main
+- Update Popup methods #addClass & #removeClass to return instance of Popup ([#3975](https://github.com/maplibre/maplibre-gl-js/pull/3975))
```
Comment (HarelM) [inline CHANGELOG.md:2]:
```suggestion
- Update Popup methods `addClass` & `removeClass` to return instance of Popup ([#3975](https://github.com/maplibre/maplibre-gl-js/pull/3975))
```

**Review by HarelM — APPROVED** [review]

**Pull request merged**